836-tests.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
```

Comment:
Could you convert this file into Python tests? Here is an example of a FIL test using SKLearn: https://github.com/rapidsai/cuml/blob/branch-0.13/python/cuml/test/test_fil.py#L189